scenarios/test_codegen/test/SerDe_Test.res

```diff
@@ -101,4 +101,83 @@ describe("SerDe Test", () => {
     | _ => Assert.fail("Should have returned a row")
     }
   })
+
+  Async.it("All type entity without array types for unnest case", async () => {
```

Comment:
It would be nice to see a test showing the sql query that gets built on a given entity

Reply:
It shouldn't be difficult. I need to split the code a little bit.